storehaus-hbase/src/main/scala/com/twitter/storehaus/asynchbase/AsyncHBaseStore.scala

```diff
@@ -20,7 +20,6 @@ import com.twitter.util.{Future, FuturePool}
 import java.util.concurrent.Executors
 import org.hbase.async.{DeleteRequest, PutRequest, GetRequest, HBaseClient}
 import com.twitter.bijection.{Injection, Codec, Conversion}
-import scala.Some
 import Conversion._
 import scala.collection.JavaConverters._
 
```

Comment:
I wonder if what we really want is a
```scala
case class Table(asBytes: Array[Byte])
case class Column(asBytes: Array[Byte])
case class ColumnFamily(asBytes: Array[Byte])
case class Key(asBytes: Array[Byte])
case class Result(timestamp: Long, valueAsBytes: Array[Byte])
class AsyncHBaseStore extends Store[(Table, ColumnFamily, Column, Key), List[Result]]
```
and then if you just want the change the key you can do:
```scala
val baseKey = (tab, cf, col, Key(Array[Byte]()))
val store: AsyncHBaseStore = ...
def get(k: Array[Byte]) = store.get(baseKey.copy(_4 = Key(k)))`
```
or something like this. Not sure why we should keep the table, column, etc... out of the key if people want to use that?